### PR TITLE
Problem: (fix #85) STATE-SYNC configuration is not updated

### LINKED ISSUE #85: Problem: STATE-SYNC configuration is not updated

### PR DESCRIPTION
**Pull request by allthatjazzleo**
Solution: update STATE-SYNC configuration

### CODE REVIEW

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -127,9 +127,10 @@ Follow the below optional steps to enable state-sync.
   $ sed -i.bak -E "s|^(enable[[:space:]]+=[[:space:]]+).*$|\1true| ; \
   s|^(rpc_servers[[:space:]]+=[[:space:]]+).*$|\1\"https://testnet-croeseid.crypto.com:26657,https://testnet-croeseid.crypto.com:26657\"| ; \
   s|^(trust_height[[:space:]]+=[[:space:]]+).*$|\1$BLOCK_HEIGHT| ; \
-  s|^(trust_hash[[:space:]]+=[[:space:]]+).*$|\1\"$TRUST_HASH\"|" ~/.chain-maind/config/config.toml
+  s|^(trust_hash[[:space:]]+=[[:space:]]+).*$|\1\"$TRUST_HASH\"| ; \
+  s|^(seeds[[:space:]]+=[[:space:]]+).*$|\1\"\"| ; \
+  s|^(persistent_peers[[:space:]]+=[[:space:]]+).*$|\1\"b2c6657096aa30c5fafa5bd8ced48ea8dbd2b003@52.76.189.200:26656,ef472367307808b242a0d3f662d802431ed23063@175.41.186.255:26656,d3d2139a61c2a841545e78ff0e0cd03094a5197d@18.136.230.70:26656\"|" ~/.chain-maind/config/config.toml
```
Comment (yihuang) [inline docs/getting-started/croeseid-testnet.md:132]: How about just set `persistent_peers` by default?

Comment (allthatjazzleo) [reply docs/getting-started/croeseid-testnet.md:132]: I think both is ok
what do you think?@lezzokafka

Comment (lezzokafka) [reply docs/getting-started/croeseid-testnet.md:132]: Yes, both are okay. 
Let's do `persistent_peers` by default.

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**